src/Microsoft.AspNetCore.OData/Batch/ODataBatchMiddleware.cs

```diff
@@ -34,13 +34,15 @@ public ODataBatchMiddleware(RequestDelegate next)
         /// <returns>A task that can be awaited.</returns>
         public async Task Invoke(HttpContext context)
         {
-            // Attempt to match the path to a bach route.
+            HttpRequest request = context.Request;
+            bool isPreFlight = request.Method.Equals("Options", StringComparison.OrdinalIgnoreCase);
```

Comment:
Why not use `bool isPreFlight = HttpMethods.IsOptions(request.Method)`?

Reply:
Updated.

Reply:
Updated